clustered/server/src/main/java/org/ehcache/clustered/server/EhcachePassiveEntity.java

```diff
@@ -43,11 +43,13 @@
 import org.terracotta.entity.PassiveServerEntity;
 import org.terracotta.entity.ServiceRegistry;
 import org.terracotta.offheapresource.OffHeapResources;
+import org.terracotta.statistics.StatisticsManager;
 
 import java.util.Collections;
 import java.util.Map;
 import java.util.Set;
 import java.util.UUID;
+import org.terracotta.context.TreeNode;
```

Comment:
I had made some changes here, but they have been reverted.  I guess I missed an import.  I will remove it.